### PR TITLE
fixes issue 482 flattened keys not exporting in nested format properly.

### PR DESCRIPTION
**Pull request by joeboyscout04**
Fixes [Issue-482](https://github.com/3lvis/Sync/issues/482) flattened objects not exporting in nested format.

### CODE REVIEW

**Comment by joeboyscout04** [comment]
Can I get a maintainer to merge this one ?

File: Source/PropertyMapper/PropertyMapper.m [hunk 1]
```diff
@@ -134,6 +134,29 @@ - (void)hyp_setDictionaryValue:(id)value forKey:(NSString *)key
                              andRelationshipType:relationshipType];
 }
 
+- (NSDictionary<NSString *, id> *)buildDbObjectWithFieldName:(NSString *)fieldName fieldValue:(id)fieldValue object:(NSMutableDictionary *)incoming {
```
Comment (3lvis) [inline Source/PropertyMapper/PropertyMapper.m]: Doesn't seem like this method is used. 🤔

**Comment by 3lvis** [comment]
Hi @joeboyscout04,

Sorry for not being so active, I think there's an unused method. Could you verify please? Besides that this is looking good, thanks for your help!

**Comment by joeboyscout04** [comment]
Whoops, got a bit carried away there.  It's gone now.

**Comment by 3lvis** [comment]
👏 Thanks!

**Pull request merged**